### ISSUE TITLE
Add more possible questions to the skill

### ISSUE THREAD

**Issue by saurabhjn76** [fossasia/susi_skill_data#119]
Currently, the skills have very limited questions, to improve skills we can add more questions which bring out the same answer and covers other possibilities.

**Would you like to work on the issue?**

This should be solved collectively.

**Comment by hpdang** [comment]
@saurabhjn76 you are right that we need many possible questions for each skill we create. But this can't be an issue. Please specify the skill that needed more work.

**Comment by saurabhjn76** [comment]
- **assitants**
    - language_translation.txt
- **Entertainment** 
   - creator_info
   - draw_a_card
   - flip coin
   - horoscope
   - jokes
  - poem 
  - trvia question
  - bitcoin

These are the few skills in which few questions could be added, I will keep updating the list.

**Comment by hpdang** [comment]
@saurabhjn76 so your focus of Milestone 6 would be extend the questions of existing skills. Please also keep in mind sometimes it doesn't need to be a question (e.g tell me....) and also pay attention to the grammar you know what time it is instead of what is the time?), sometimes people/users make grammar mistakes (e.g how do he know instead of how does he know)

**Comment by saurabhjn76** [comment]
@hpdang thanks for the guidance, I will frame queries keeping above things in mind. :+1:
